scripts/globals/summon.lua

```diff
@@ -1,241 +1,187 @@
-    fstr = avatarFSTR(avatar:getStat(dsp.mod.STR), target:getStat(dsp.mod.VIT));
+function AvatarPhysicalMove(avatar,target,skill,numberofhits,accmod,dmgmod,dmgmodsubsequent,tpeffect,mtp100,mtp200,mtp300)
 
-    lvluser = avatar:getMainLvl();
-    lvltarget = target:getMainLvl();
-    local master = avatar:getMaster();
-    local bonusacc = utils.clamp(master:getSkillLevel(dsp.skill.SUMMONING_MAGIC) - master:getMaxSkillLevel(avatar:getMainLvl(), dsp.job.SMN, dsp.skill.SUMMONING_MAGIC), 0, 200);
-    acc = avatar:getACC() + bonusacc;
-    eva = target:getEVA();
+    local returninfo = {}
 
-    local base = avatar:getWeaponDmg() + fstr;
-    local ratio = avatar:getStat(dsp.mod.ATT)/target:getStat(dsp.mod.DEF);
+    local acc = avatar:getACC() + utils.clamp(getSummoningSkillOverCap(avatar:getMaster()), 0, 200)
+    local eva = target:getEVA()
+    local dmg = avatar:getWeaponDmg()
+    local minFstr, maxFstr = avatarFSTR(avatar:getStat(dsp.mod.STR), target:getStat(dsp.mod.VIT))
+    local ratio = avatar:getStat(dsp.mod.ATT) / target:getStat(dsp.mod.DEF)
 
-    lvldiff = lvluser - lvltarget;
+    -- Note: Avatars do not have any level correction. This is why they are so good on Wyrms! // https://kegsay.livejournal.com/tag/smn!
+    local hitrate = utils.clamp(acc - eva, 20, 95)
 
-    -- work out hit rate for mobs (bias towards them)
-    hitrate = (acc*accmod) - eva;
-    if (lvluser > lvltarget) then
-        hitrate = hitrate + ((lvluser-lvltarget)*5);
-    end
-    if (lvltarget > lvluser) then
-        hitrate = hitrate + ((lvltarget-lvluser)*3);
-    end
-    if (hitrate > 95) then
-        hitrate = 95;
-    end
-    if (hitrate < 20) then
-        hitrate = 20;
-    end
+    -- add on native crit hit rate (guesstimated, it actually follows an exponential curve)
+    local critrate = (avatar:getStat(dsp.mod.DEX) - target:getStat(dsp.mod.AGI)) * 0.005 -- assumes +0.5% crit rate per 1 dDEX
+    critrate = critrate + avatar:getMod(dsp.mod.CRITHITRATE) / 100
+    critrate = utils.clamp(critrate, 0.05, 0.2);
 
-    if (base < 1) then
-        base = 1;
-    end
-    hitdamage = base * dmgmod1;
-    subsequenthitdamage = base * dmgmodsubsequent;
-    if (ratio<=1) then
-        maxRatio = 1;
-        minRatio = 1/3;
-    elseif (ratio<1.6) then
-        maxRatio = ((4/6) * ratio) + (2/6);
-        minRatio = ((7/9) * ratio) - (4/9);
-    elseif (ratio<=1.8) then
-        maxRatio = 1.8;
-        minRatio = 1;
-    elseif (ratio<3.6) then
-        maxRatio = (2.4 * ratio) - 2.52;
-        minRatio = ((5/3) * ratio) - 2;
+    -- Applying pDIF
+    if (ratio <= 1) then
+        maxRatio = 1
+        minRatio = 1/3
+    elseif (ratio < 1.6) then
+        maxRatio = ((2/3) * ratio) + (1/3)
+        minRatio = ((7/9) * ratio) - (4/9)
+    elseif (ratio <= 1.8) then
+        maxRatio = 1.8
+        minRatio = 1
+    elseif (ratio < 3.6) then
+        maxRatio = (2.4 * ratio) - 2.52
+        minRatio = ((5/3) * ratio) - 2
```

Comment:
Remove all the parentheses in this block. None of them are necessary. Order of operations handle it correctly. Outer parentheses in conditionals are also against current style-guide.

Also space out numbers and operators.